Model/Product/Service.php

```diff
@@ -321,7 +325,7 @@ public function processForAccount(array $uniqueProductIds, Store $store, Account
         $productIdsStillExist = [];
 
         if (!empty($productsStillExist)) {
-            $op = new UpsertProduct($nostoAccount);
+            $op = new UpsertProduct($nostoAccount, $this->nostoHelperUrl->getActiveDomain());
```

Comment:
Shouldn't this affect all operations including update order, update settings etc?

Reply:
I am checking the other operations that need to be updated.